NAMESPACE

```diff
@@ -1,36 +1,5 @@
 # Generated by roxygen2: do not edit by hand
 
-export(R_to_growth)
-export(adjust_infection_to_report)
-export(bootstrapped_dist_fit)
```

Comment:
Ah - removing this is what is causing the build to fail. This has happened during the failed compilation I think. I will clean up and merge